roles/confluent.kafka_connect/defaults/main.yml

```diff
@@ -23,6 +23,20 @@ kafka_connect_default_internal_replication_factor: "{{ [ groups['kafka_broker']
 
 kafka_connect_group_id: connect-cluster
 
+kafka_connect_plugins_path: 
+- /usr/share/java
+kafka_connect_confluent_hub_install: "{{true if kafka_connect_confluent_hub_plugins|length >0 else false}}" 
+kafka_connect_confluent_hub_client:
+  file: "http://client.hub.confluent.io/confluent-hub-client-latest.tar.gz"
```

Comment:
https?

Reply:
It was the link published on the website.  I'll check if HTTPS is supported by the endpoint.

Reply:
Looks like http is the only supported route at this time. 😞

Reply:
looks like a HEAD request times out, so probably not